### DISCUSSION TITLE
大家可以多多禮讓行人嗎

### DISCUSSION THREAD

**Discussion by Jamesmoneya** [1101-GS4719-C/Python-Community-GS4719-C#1435] [General]
我好多次右轉停下來讓路人，結果被後面的車按喇叭..
還有一次經過沒有紅綠燈的斑馬線停下來讓一個阿伯過馬路，被後方機車按喇叭，他通過時還語帶嘲諷的說哈嘍你在衝三小~~

只能說沒素質到人太多了，想讓也不行

**Comment by 111607004** [comment]
嗚嗚辛苦你了，但你是對的，本來就該禮讓行人啊！
我覺得在中壢這種情況很常發生，這大概是每天上下班時間火車站前會大塞車的原因吧！每台車都橫衝直撞，人也橫衝直撞，最後全部卡在一起就大塞車了...

**Comment by brian431** [comment]
台灣人真的不怎麼喜歡禮讓行人。反觀我在歐洲去的所有國家都非常禮讓行人。就連我離路口還有一段距離車子都會直接停下來等我走過去，害我非常不習慣。

**Comment by Elaine2899** [comment]
在台灣，行人不管是過馬路還是走在路邊真的都要很小心，騎車開車的也是，之前在路上，有路邊亂停車的人突然開車門，害我差點撞上他，超級危險！希望台灣在教育環節以及考駕照的時候都可以把這些基本的用路素質教好，很多人都是邊騎／開車邊學怎麼馬路上生存，我自己也是，但不應該讓生命暴露在這樣危險的環境之下

**Comment by pudding0803** [comment]
我是行人的時候被讓反而很不習慣，因為很怕被撞所以平常都是讓車子🥲，希望哪天可以改變社會大眾的習慣，雖然現在好像有法律規定？

但行人過馬路也要專心就是了😌

**Comment by PaulSu0905** [comment]
我覺得開車騎車就不是一件應該很急求快的事情，但許多台灣的駕駛人好像都把自己與他人生命安全的重要性擺在他的時間重要性之後。

**Comment by Brownnnn78** [comment]
雖然行人在台灣真的很卑微，（可能是因為我們的聲音沒有喇叭大聲？）但是有時看到會禮讓行人的駕駛們耐心的等待反而碰到一直滑手機慢慢過馬路的人，我都想去推他一把，只能說台灣交通上的素質真的需要改進！有可能不禮讓行人的駕駛下車後就變成罵台灣行人地獄的人了。

**Comment by Evening218** [comment]
我常常在網路上看到別人說台灣大概是唯一一個會看到行人禮讓車輛的地方,但是我認為在大部分車輛都不禮讓的情況下,不要說是行人太卑微,只能說行人為了自己的人身安全,本來就不可能真的去跟車輛硬槓,所以這個狀況大概真的很難改變了

**Comment by ashleyy-huang** [comment]
我認為許多人都沒有禮讓行人的觀念，總是仗著自己是機車或著車子而急著要過去，最近在台灣被說是行人地獄之後，可以發現大家開始乖住且重視這個議題，也可以發現馬路上的人開始回停下來讓行人先過，希望這個現象可以繼續保持來保護行人路權

**Comment by Chiuhsianglin** [comment]
台灣在禮讓行人這個地方的確還要再進步，經過路口的時候可能要注意一下速度，後面的車也要注意，不要不注意就撞上去了。

**Comment by bestbanana295** [comment]
沒錯，有時候在網路上面看到有人再讓行人的時候背後八的影片就很萊契，希望以後大家可以有多一點的意識，保持行車距離，打造更好的交通鐶境。不要讓別人看傚話了，我覺得最重要的就是駕駛都有一個守法的心態，不要鑽漏洞，耍小聰明。

**Comment by aa103967622** [comment]
停下來禮讓路人絕對是常識，後面的車子沒有跟著禮讓就算了，還反過來嘲諷，三寶就是這樣，沒有好好培養自己的道路觀就跑出來禍害人。

**Comment by zhiron544** [comment]
我也有過類似的經歷，停下來讓行人過馬路是一種基本的禮貌和法律規定，但是有些駕駛者卻不理解或不尊重。我覺得很不負責任
